src/Plaid/Management/PlaidLinkResponse.cs

```diff
@@ -26,18 +26,40 @@ public class PlaidLinkResponse
 		/// </summary>
 		/// <value>The accounts.</value>
 		[JsonProperty("accounts")]
+		[System.Diagnostics.CodeAnalysis.SuppressMessage(
+			"Performance",
+			"CA1819:Properties should not return arrays",
+			Justification = "Matching the data structure from the Plaid API")]
 		public AccountInfo[] Accounts { get; set; } = null!;
```

Comment:
We should be able to switch to `IReadOnlyList<AccountInfo>`. This is probably the correct solution for response objects.

Reply:
I can just take out that suppression message for now - I added it for my own code clarity and didn't clean it when copying over the code to this project. I'm not sure making the change to IReadOnlyList in one case is good - if that change is happening, it should be done as a sweep across the code.

Reply:
Yeah, let's do a separate PR for sweeping change to `IReadOnlyList<>`. Thanks.